sklearn/utils/_indexing.py

```diff
@@ -451,6 +460,9 @@ def resample(*arrays, replace=True, n_samples=None, random_state=None, stratify=
         If not None, data is split in a stratified fashion, using this as
         the class labels.
 
+    sample_weight : array-like of shape (n_samples,) or None, default=None
+        Contains weight values to be associated with each sample.
```

Comment:
We need to explain how those weights impact resampling: they are normalized to sum to one and interpreted as probability for sampling each data point with replacement.

Reply:
We also need to add the `.. versionadded:: 1.6` directive to this entry of the docstring. Look for other examples in the code base for usage examples of this directive in the "Parameters" section of docstrings.


```
$ git grep -C 5 versionadded sklearn
```